### DISCUSSION TITLE
[1292940] Cube Escape Collection 逃离方块

### DISCUSSION THREAD

**Discussion by shaoqi-li** [pjy612/SteamManifestCache#205] [Share]
`appid:1292940`
keys:
```
1292941;284d467ea0207a437d408350b2d125920ed0671c3f156321d4dda103aa05c0bc
```
[1292940.zip](https://github.com/pjy612/SteamManifestCache/files/13796880/1292940.zip)
内测码: 287EE47F7317F947
email: vimoe10@gmail.com

**Comment by pjy612** [comment]
？有啥区别吗？

**Reply by shaoqi-li** [reply]
我没看到你的回复和你发的邮件，以为你没看到，就又发了一遍

**Reply by ZEERDEER** [reply]
有可能是不想回复😁，而且你分享的游戏本身库里面就有。

**Reply by shaoqi-li** [reply]
好吧，我看分享贴里没有就传了这个

**Reply by pjy612** [reply]
要不查查 垃圾邮箱？
![image](https://github.com/pjy612/SteamManifestCache/assets/4072526/4cc6fa07-80c8-48b5-b18a-215eadee4f22)

**Reply by shaoqi-li** [reply]
没想到被Google丢到垃圾邮件里了，而steam和epic的推销广告一个都没屏蔽掉，感谢作者
